packages/xrpl/src/models/methods/nftsByIssuer.ts

```diff
@@ -0,0 +1,68 @@
+import { NFToken } from '../common'
+
+import { BaseRequest, BaseResponse, LookupByLedgerRequest } from './baseMethod'
+
+/**
+ * The nfts_by_issuer method returns a list of NFTokens issued by the account.
+ * The order of the NFTs is not associated with the date the NFTs were minted.
+ * Expects a response in the form of a {@link
+ * NFTsByIssuerResponse}.
+ *
+ * @category Requests
+ */
+export interface NFTsByIssuerRequest
```

Comment:
This interface should be imported into the index file of methods, then client to be available for public use and matched with the response. Please follow the same steps of other methods (i.e `AccountChannels`)